### PR TITLE
Update data.yml

### PR DESCRIPTION
**Pull request by wallace**

### CODE REVIEW

File: data.yml [hunk 1]
```diff
@@ -578,6 +578,18 @@ people:
   - company: Stitch Fix
     position: Principal Engineer
     start_day: 2016-10-17
+    end_day: 2017-01-31
+  - company: Georgia General Assembly
+    position: House Representative for District 119
+    start_day: 2017-11-??
```
Comment (ckdake) [inline data.yml]: maybe do 01 in here instead of ??. Not sure this is going to work the way you want!

**Review by ckdake — CHANGES_REQUESTED** [review]

File: data.yml (hunk 1 above)
Comment (wallace) [reply data.yml]: 😆😆

This is all a placeholder so I can get the dates right and get the PR ready for a merge!

Comment (wallace) [inline data.yml]:
```suggestion
    start_day: 2017-11-27
```

https://ballotpedia.org/Jonathan_Wallace_(Georgia)

File: data.yml [hunk 2]
```diff
@@ -578,6 +578,18 @@ people:
...
+    end_day: 2019-01-13??
+  - company: Softgiving
+    position: VP of Engineering
+    start_day: 2017-03-01
+    end_day:
+  - company: GitHub
+    position: Senior Software Engineer
+    start_day: 2019-06-19??
```
Comment (wallace) [inline data.yml]:
```suggestion
    start_day: 2019-06-18
```

File: data.yml [hunk 3]
```diff
@@ -578,6 +578,18 @@ people:
   - company: Stitch Fix
     position: Principal Engineer
     start_day: 2016-10-17
+    end_day: 2017-01-31
+  - company: Georgia General Assembly
+    position: House Representative for District 119
+    start_day: 2017-11-27
+    end_day: 2019-01-13??
```
Comment (wallace) [inline data.yml]:
```suggestion
    end_day: 2019-01-13
```

**Review by ckdake — APPROVED** [review]

File: data.yml [hunk 4]
```diff
@@ -578,6 +578,18 @@ people:
...
     start_day: 2016-10-17
+    end_day: 2017-01-31
+  - company: Georgia General Assembly
+    position: House Representative for District 119
+    start_day: 2017-11-27
+    end_day: 2019-01-13
+  - company: Softgiving
+    position: VP of Engineering
+    start_day: 2017-03-01
+    end_day:
```
Comment (wallace) [inline data.yml]:
```suggestion
    end_day: 2019-06-07
```

**Review by wallace — COMMENTED** [review]
All done!

**Comment by ckdake** [comment]
I don't know that I believe you

**Pull request merged**